src/ducks/balance/Balance.jsx

```diff
@@ -241,13 +241,15 @@ class Balance extends PureComponent {
   }
 
   updateQueries() {
-    this.props.accounts.fetch().then(resp => {
+    const { accounts, groups, transactions, triggers } = this.props
+
+    accounts.fetch?.().then(resp => {
       if (resp.meta.count > 0) {
-        this.props.groups.fetch()
+        groups.fetch?.()
       }
     })
-    this.props.transactions.fetch()
-    this.props.triggers.fetch()
+    transactions.fetch?.()
+    triggers.fetch?.()
```

Comment:
I am not sure this is the right fix, why would the rersult of queryConnect not have fetch() ? It seems like a bug of cozy-client and am not sure that the app should dirty fix it.

Reply:
Aucune idée, je suis tombé dessus par hasard en testant le service, j'ai vu ça dans les log et en regardant ultra rapidement ce que Balance faisait j'ai ajouté ça mais je n'ai pas remonté la piste plus haut dans Client vu que c'était pas le sujet de la PR. J'ai fait ça en "quick win" mais on peut faire une issue sur Client

Reply:
Ca complexifie le code, et ca perturbe la lecture. Dnas ce cas là, je lis le code et je me dis, "tiens pourquoi ya besoin de cette condition, est ce que je dois rajouter ce check à tous mes autres codes ?" et si je vais pas checker le code, je vais répliquer la condition et rapidement tout le monde rajoute cette condition parce qu'on comprend pas ce qui se passe.

Reply:
Ca veut aussi dire que les autres applis ne bénéficient pas du fix.

Reply:
ok je l'ai sorti, je vais investiguer dessus, je ferai une issue si j'aboutis pas